packages/ui-react/src/components/UserMenu/UserMenu.tsx

```diff
@@ -63,74 +65,74 @@ const UserMenu = ({
   }, [onClick, navigate, accountController]);
 
   return (
-    <ul onFocus={onFocus} onBlur={onBlur} className={styles.menuItems}>
-      {profilesEnabled && selectProfile && (
-        <ProfilesMenu
-          profiles={profiles ?? []}
-          currentProfile={currentProfile}
-          small={small}
-          selectingProfile={!!isSelectingProfile}
-          selectProfile={selectProfile}
-          createButtonLabel={t('nav.add_profile')}
-          switchProfilesLabel={t('nav.switch_profiles')}
-          onCreateButtonClick={() => navigate(PATH_USER_PROFILES_CREATE)}
-        />
-      )}
-      <li className={styles.sectionHeader}>{t('nav.settings')}</li>
-      {profilesEnabled && currentProfile && (
-        <li>
-          <MenuButton
+    <>
+      <h2 className={styles.sectionHeader} id={titleId}>
+        {t('nav.settings')}
+      </h2>
+      <ul onFocus={onFocus} onBlur={onBlur} className={styles.menuItems}>
+        {profilesEnabled && selectProfile && (
+          <ProfilesMenu
```

Comment:
Inside the <ProfilesMenu />, there are also list items and a divider 😅

Reply:
I think i will need another 2 hours to optimize the `ProfilesMenu`. It also got a heading and a divider.
I want to split them in to 2 lists with each their own heading. But it also got `onFocus`/`onBlur` handlers for the fly-out menu, which I want to reevaluate after restructure, if they work correctly.

Can you tell me an appconfig ID which has profiles enabled? With your approval, I will fix it separately.

Reply:
The profiles feature has been enabled, but we can do it separately from this PR 👍